packages/server/lib/server.coffee

```diff
@@ -38,6 +39,13 @@ templateEngine = require("./template_engine")
 DEFAULT_DOMAIN_NAME    = "localhost"
 fullyQualifiedRe       = /^https?:\/\//
 
+NON_PROXY_PATH_WHITELIST = [
```

Comment:
maybe call this: `ALLOWED_PROXY_BYPASS_WHITELIST` if you think that better describes this which turns this into an affirmative